app/services/client/popup.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+class service {
+  getWindowSize() {
+    const dualScreenLeft = window.screenLeft !== undefined ? window.screenLeft : screen.left,
+      dualScreenTop = window.screenTop !== undefined ? window.screenTop : screen.top,
+      width = window.innerWidth || document.documentElement.clientWidth || screen.width,
+      height = window.innerHeight || document.documentElement.clientHeight || screen.height;
+
+    return {dualScreenLeft, dualScreenTop, height, width};
```

Comment:
Please mind the spacing
```suggestion
    return { dualScreenLeft, dualScreenTop, height, width };
```

Reply:
👍